FSharpLu.Json/Scripts/load-project-debug.fsx

```diff
@@ -0,0 +1,9 @@
+// Warning: generated file; your changes could be lost when a new file is generated.
```

Comment:
They get generated by the Visual F# Power Tools plugin for Visual Studio. I would add them to .gitignore and remove the corresponding commit (with `git rebase -i HEAD~3`)